webpack/react_app/components/TargetingHosts/TargetingHostsPage.scss

```diff
@@ -0,0 +1,3 @@
+.targeting-hosts-pagination {
+  margin-top: -7px;
+}
```

Comment:
can you also add `min-height: 350px` to the page wrapper?
so it will be easy to see the table, and it won't jump when the searchbar opens:

![Selection_335](https://user-images.githubusercontent.com/26363699/86786061-32d45a80-c06c-11ea-950a-ef726cf504ec.png)

page ends here :arrow_up: